OWML.Common/IModConfig.cs

```diff
@@ -16,7 +16,6 @@ public interface IModConfig
         [Obsolete("Use GetSettingsValue instead")]
         T GetSetting<T>(string key);
 
-        void MakeConsistentWithDefaults(IModConfig defaultConfig);
         void ResetToDefaults(IModConfig defaultConfig);
```

Comment:
Not sure if stuff should be moved to ModData, but I'm sure these two methods belong together.

Reply:
but ModData is not accessible in mods

Reply:
not sure why mods would want to reset their own settings themselves though

Reply:
moving it to ModData would noticeably simplify ModConfigMenu.OnReset though

Reply:
@Raicuparta what do you think? Should I remove ResetToDefaults from ModConfig?

Reply:
No opinion here 🤷‍♂️